mods/cnc/maps/funpark01/map.yaml

```diff
@@ -0,0 +1,542 @@
+		Location: 46,38
+		Owner: Neutral
+	Actor38: tc04
+		Location: 45,42
+		Owner: Neutral
+	Actor39: t01
+		Location: 54,46
+		Owner: Neutral
+	Actor40: t01
+		Location: 51,49
+		Owner: Neutral
+	Actor41: t01
+		Location: 14,47
+		Owner: Neutral
+	Actor42: t02
+		Location: 20,49
+		Owner: Neutral
+	Actor43: t01
+		Location: 25,53
+		Owner: Neutral
+	Actor44: t03
+		Location: 53,47
+		Owner: Neutral
+	Actor45: t05
+		Location: 40,49
+		Owner: Neutral
+	Actor46: t06
+		Location: 43,48
+		Owner: Neutral
+	Actor47: tc04
+		Location: 37,47
+		Owner: Neutral
+	Actor48: tc04
+		Location: 29,51
+		Owner: Neutral
+	Actor49: tc02
+		Location: 22,52
+		Owner: Neutral
+	Actor50: tc01
+		Location: 21,47
+		Owner: Neutral
+	Actor51: tc01
+		Location: 47,44
+		Owner: Neutral
+	Actor52: tc01
+		Location: 47,49
+		Owner: Neutral
+	Actor53: t07
+		Location: 52,42
+		Owner: Neutral
+	Actor54: t07
+		Location: 45,46
+		Owner: Neutral
+	Actor55: t07
+		Location: 41,47
+		Owner: Neutral
+	Actor56: t06
+		Location: 47,46
+		Owner: Neutral
+	Actor57: t01
+		Location: 49,43
+		Owner: Neutral
+	Actor58: t01
+		Location: 52,29
+		Owner: Neutral
+	Actor59: tc04
+		Location: 41,27
+		Owner: Neutral
+	Actor60: t01
+		Location: 13,7
+		Owner: Neutral
+	Actor61: t03
+		Location: 18,10
+		Owner: Neutral
+	Actor62: t03
+		Location: 20,14
+		Owner: Neutral
+	Actor63: tc04
+		Location: 12,9
+		Owner: Neutral
+	Actor64: tc01
+		Location: 15,14
+		Owner: Neutral
+	Actor65: tc02
+		Location: 20,11
+		Owner: Neutral
+	Actor66: t05
+		Location: 12,14
+		Owner: Neutral
+	Actor67: tc01
+		Location: 16,8
+		Owner: Neutral
+	Actor68: t07
+		Location: 19,8
+		Owner: Neutral
+	Actor69: t07
+		Location: 17,12
+		Owner: Neutral
+	Actor70: t07
+		Location: 22,10
+		Owner: Neutral
+	Actor71: t07
+		Location: 29,11
+		Owner: Neutral
+	Actor72: t08
+		Location: 33,14
+		Owner: Neutral
+	Actor73: t02
+		Location: 27,17
+		Owner: Neutral
+	Actor74: t02
+		Location: 47,7
+		Owner: Neutral
+	Actor75: t01
+		Location: 44,6
+		Owner: Neutral
+	Actor76: t01
+		Location: 40,10
+		Owner: Neutral
+	Actor77: t03
+		Location: 37,8
+		Owner: Neutral
+	Actor78: t05
+		Location: 46,12
+		Owner: Neutral
+	Actor79: tc01
+		Location: 33,20
+		Owner: Neutral
+	Actor80: tc01
+		Location: 50,21
+		Owner: Neutral
+	Actor81: tc01
+		Location: 28,16
+		Owner: Neutral
+	Actor82: tc01
+		Location: 48,14
+		Owner: Neutral
+	Actor83: tc01
+		Location: 42,10
+		Owner: Neutral
+	Actor84: tc02
+		Location: 33,18
+		Owner: Neutral
+	Actor85: tc02
+		Location: 39,13
+		Owner: Neutral
+	Actor86: tc02
+		Location: 36,5
+		Owner: Neutral
+	Actor87: tc02
+		Location: 50,9
+		Owner: Neutral
+	Actor88: tc02
+		Location: 52,16
+		Owner: Neutral
+	Actor89: tc04
+		Location: 38,18
+		Owner: Neutral
+	Actor90: tc04
+		Location: 31,12
+		Owner: Neutral
+	Actor91: tc04
+		Location: 45,9
+		Owner: Neutral
+	Actor92: tc04
+		Location: 43,16
+		Owner: Neutral
+	Actor93: v01
+		Location: 16,10
+		Owner: Civilian
+		Health: 1
+		Facing: 0
```

Comment:
We are currently removing those default values. https://github.com/OpenRA/OpenRA/pull/7319